### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by lmorchard**
Plus, one small bugfix for missing parameters.

### CODE REVIEW

File: tests/unit/test.index.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      return updateCallback(null, 'updated ' + this.path);
+    };
+    var stylesheets = [
+      new MockStyleSheet('css/style1.css', 'old contents 1'),
+      new MockStyleSheet('css/style2.css', 'old contents 2'),
+      new MockStyleSheet('css/style3.css', 'old contents 3')
+    ];
+
+    // Set up some mock files intended to replace stylesheet contents
+    var FILES = {};
```
Comment (tofumatt) [inline tests/unit/test.index.js]: Why SO LOUD?

File: tests/unit/test.index.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    }).then(function(results) {
+      test.ok(false);
+      test.done();
+    }).catch(function(err) {
+      test.done();
+    });
+  },
+
+
+  'reloadCSS() should make Firefox API calls to reload CSS': function(test) {
```
Comment (tofumatt) [inline tests/unit/test.index.js:47]: The placement of variable definitions and mixed use of `regularVars` and `ALL_CAPS` variables in this function has me a bit confused. I'm not sure why certain things are treated like `CONSTANTS` when they're all local to this function's scope anyway.

**Comment by tofumatt** [comment]
r+wc, just curious about the variables as mentioned.

**Comment by lmorchard** [comment]
Yeah, no good reason for the ALL_CAPS. Most of them were constants at the top of the file, at one point. But, then I moved them into the test function body.

**Comment by sole** [comment]
Y U no merge? oh wait, conflicts!

**Comment by tofumatt** [comment]
Oh, yeah, sorry, I usually mean "feel free to merge after comments are addressed" when I say r+wc but that is hardly clear, sorry.

I'll see if I can resolve these now and merge.

**Comment by lmorchard** [comment]
Yeah, these tests have bitrotted a bit after #4 merged. Need some tweaks to account for the `tag.attach()` call

**Comment by lmorchard** [comment]
Superseded by #6

**Pull request closed without merging**